src/librustdoc/config.rs

```diff
@@ -192,6 +192,9 @@ pub struct RenderOptions {
     /// If present, playground URL to use in the "Run" button added to code samples generated from
     /// standalone Markdown files. If not present, `playground_url` is used.
     pub markdown_playground_url: Option<String>,
+    /// If false, the `select` element to have search filtering by crates on rendered docs
+    /// won't be generated.
+    pub generate_filter_crates: bool,
```

Comment:
We should also change this to something like `generate_search_filter` to match the CLI flag a little closer.